rest_framework_simplejwt/token_blacklist/models.py

```diff
@@ -3,7 +3,7 @@
 
 
 class OutstandingToken(models.Model):
-    user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, null=True, blank=True)
+    user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.SET_NULL, null=True, blank=True)
```

Comment:
If this happens then you should change the SET_NULL in the migrations (deliberately without creating a new file).

Reply:
Also, I'm not entirely sure if you should change it to be SET_NULL...